packages/ui/fe-bundle/src/pages/[host]/[[...slug]].tsx

```diff
@@ -20,40 +21,40 @@ export declare namespace Docs {
     export interface Props {
         docs: FernRegistryDocsReadV2.LoadDocsForUrlResponse;
         resolvedUrlPath: ResolvedUrlPath;
-        typographyStyleSheet?: string;
+        typographyStyleSheet: string | undefined;
+        backgroundImageStyleSheet: string | undefined;
         nextPath: ResolvedUrlPath | null;
         previousPath: ResolvedUrlPath | null;
     }
 }
 
-function Typography({ stylesheet }: { stylesheet: string }) {
+function StyleSheets({ stylesheets }: { stylesheets: string[] }) {
+    const mergedSheet = stylesheets.join("\n");
     return (
         // eslint-disable-next-line react/no-unknown-property
-        <style id="fern-font-stylesheet" jsx global>
-            {stylesheet}
+        <style id="fern-stylesheets" jsx global>
+            {mergedSheet}
         </style>
     );
 }
 
 export default function Docs({
     docs,
     typographyStyleSheet,
+    backgroundImageStyleSheet,
     resolvedUrlPath,
     nextPath,
     previousPath,
 }: Docs.Props): JSX.Element {
+    const stylesheets = [typographyStyleSheet, backgroundImageStyleSheet].filter((s) => s != null) as string[];
     return (
         <>
-            {typographyStyleSheet != null && <Typography stylesheet={typographyStyleSheet} />}
+            <StyleSheets stylesheets={stylesheets} />
```

Comment:
nitty but maybe just
```tsx
return (
  <>
    {typographyStyleSheet != null && <StyleSheet styleSheet={typographyStyleSheet} />}
    {backgroundImageStyleSheet != null && <StyleSheet styleSheet={backgroundImageStyleSheet} />}
    ...
```

in general I don't love casts (`as string[]`) since it's easy to run into runtime errors if the cast becomes incorrect later when the typings change

Reply:
HAha I get you